caravel/migrations/versions/41f6a59a61f2_database_options_for_sql_lab.py

```diff
@@ -0,0 +1,28 @@
+"""database options for sql lab
+
+Revision ID: 41f6a59a61f2
+Revises: 3c3ffe173e4f
+Create Date: 2016-08-31 10:26:37.969107
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '41f6a59a61f2'
+down_revision = '3c3ffe173e4f'
+
+
+def upgrade():
+    op.add_column('dbs', sa.Column('allow_ctas', sa.Boolean(), nullable=True))
+    op.add_column(
+        'dbs', sa.Column('expose_in_sqllab', sa.Boolean(), nullable=True))
+    op.add_column(
+        'dbs',
+        sa.Column('force_ctas_schema', sa.String(length=250), nullable=True))
```

Comment:
s/force_ctas_schema/ctas_schema
I would expect to have a var that starts with force_ to be boolean. Optional.